client/webserver/site/src/html/register.tmpl

```diff
@@ -35,17 +35,12 @@
     </form>
 
     {{- /* Set up the Decred wallet. Only shown if user has not connected their Decred wallet. */ -}}
-    <form class="card mx-auto my-5 bg1{{if not .WalletStep}} d-hide{{end}}" id="newWalletForm">
+    <form class="card mx-auto my-5 bg1 d-hide" id="newWalletForm">
       {{template "newWalletForm" "[[[reg_dcr_required]]]"}}
```

Comment:
A UX issue with this form is that when you get here from the Confirm Registration form by clicking Setup, there is no way back unless you complete the wallet setup.  Ideally you'd be able to "X" the form and it would take you back.